cmd/f3/run.go

```diff
@@ -18,6 +18,11 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// ID of the default manifest server which is responsible for periodically
+// publishing it on the diagnostics topic
+// TODO: Make it configurable through a flag.
+var defaultManifestServerID, _ = peer.Decode("12D3KooWHQRSDFv4FvAjtU32shQ7znz7oRbLBryXzZ9NMK2feyyH")
```

Comment:
My sense is that this should be externally configurable with no hardcoded value, i.e. if unset no manifest server is taken to consideration.